docker/compose.deploy.yaml

```
@@ -0,0 +1,267 @@
+      - source: caddy_data
+        target: /data
+        type: volume
+      - source: ./docker/Caddyfile
+        target: /etc/caddy/Caddyfile
+        type: bind
+  minio:
+    # https://docs.docker.com/compose/environment-variables/
+    # https://docs.docker.com/reference/compose-file/services/#environment
+    environment:
+      - MINIO_BROWSER=${MINIO_BROWSER:?error}
+        # https://min.io/docs/minio/linux/reference/minio-server/settings/root-credentials.html#envvar.MINIO_ROOT_PASSWORD
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)